### PR TITLE
Added NM Skill / Spell lists for Visions and Scars of Abyssea

### PR DESCRIPTION
**Pull request by R3P0FFXI**
…sea - Tahrongi, Abyssea - Attohwa,	Abyssea - Misareaux, Abyssea - Vunkerl

Added Spell lists for Abyssea Zones Abyssea - Attohwa, Abyssea - Misareaux, Abyssea - Vunkerl

Added example scripts for NM Skills in Abyssea-Konschta

### CODE REVIEW

File: sql/mob_skill_lists.sql [hunk 1]
```diff
@@ -2692,3 +2692,770 @@ INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2145); -- Gospel of
 INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2146); -- Void of Repentance "Repent thy irreverence..."
 INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2147); -- Divine Judgement   "Accept thy destruction..."
 
+-- Alkonost
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,576); -- back heel
+-- INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,579); -- choke breath
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,578); -- nihlity song
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,580); -- fantod
+-- INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,1330); -- hoof volley
```
Comment (Hozu) [inline sql/mob_skill_lists.sql]: Hoof volley has a script. https://github.com/DarkstarProject/darkstar/blob/master/scripts/globals/mobskills/Hoof_volley.lua

Or is it supposed to be different, somehow? On the other hand, Fantod does not have a script.

Comment (R3P0FFXI) [inline sql/mob_skill_lists.sql]: @Hozu I may have commented the wrong line by mistake I just put the fix in that will correct that issue.

File: sql/mob_skill_lists.sql [hunk 2]
```diff
@@ -2692,3 +2692,770 @@ INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2145); -- Gospel of
...
+INSERT INTO `mob_skill_lists` VALUES ('Grandgousier',816,658); -- catapult
+INSERT INTO `mob_skill_lists` VALUES ('Grandgousier',816,662); -- lightning roar
+INSERT INTO `mob_skill_lists` VALUES ('Grandgousier',816,1636); -- trebuchet
+
+-- Hadhayosh
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,628); -- wild horn
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,629); -- thunderbolt
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,632); -- flame armor
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,633); -- howl
+INSERT INTO `mob_skill_lists` VALUES ('Hadhayosh',817,2390); -- accursed armor
```
Comment (TeoTwawki) [inline sql/mob_skill_lists.sql]: just a note for myself for later: I think I did something custom for legiondark to get the mob's curse spikes behaving correctly, and I need to port that to darkstar still. I need to verify that and get to it if so. Last I remembered darkstars implementation of monster scripted spikes required a spikes status effect, and there is no curse spikes effect to use for this, and I am improving said implementation *anyway* so I can get it then.

Comment (TeoTwawki) [inline sql/mob_skill_lists.sql]: I think as long as there is no script this skill just get skipped in the mobs selection of tp skill anyway, so it should be fine like this. If I'm wrong we'd just need to comment it out till I fix the spike thing.

File: sql/mob_skill_lists.sql [hunk 3]
```diff
@@ -2692,3 +2692,770 @@ INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2145); -- Gospel of
 INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2146); -- Void of Repentance "Repent thy irreverence..."
 INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2147); -- Divine Judgement   "Accept thy destruction..."
 
+-- Alkonost
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,576); -- back heel
```
Comment (TeoTwawki) [inline sql/mob_skill_lists.sql]: Hey, see how the mobname is already a field in the sql line itself? I've pointed this out to you like 50 times now. Cut it out.

**Pull request merged**